pl_bolts/models/detection/yolo/yolo_module.py

```diff
@@ -0,0 +1,516 @@
+import inspect
+from typing import Dict, List, Tuple
+
+import numpy as np
+import pytorch_lightning as pl
+import torch
+import torch.nn as nn
+from torch import optim, Tensor
+
+from pl_bolts.models.detection.yolo.yolo_config import YoloConfiguration
+from pl_bolts.models.detection.yolo.yolo_layers import DetectionLayer, RouteLayer, ShortcutLayer
+from pl_bolts.optimizers.lr_scheduler import LinearWarmupCosineAnnealingLR
+from pl_bolts.utils.warnings import warn_missing_pkg
+
+try:
+    import torchvision.transforms as T
+    from torchvision.ops import nms
+    from torchvision.transforms import functional as F
+except ModuleNotFoundError:
+    warn_missing_pkg('torchvision')  # pragma: no-cover
+    _TORCHVISION_AVAILABLE = False
+else:
+    _TORCHVISION_AVAILABLE = True
+
+
+class Yolo(pl.LightningModule):
```

Comment:
Nit: I prefer `YOLO` in caps. @akihironitta What do you think?

Reply:
I agree. @Separius Would you mind following the @kaushikb11's suggestion above?

Reply:
Right, the convention seems to be to write acronyms with all caps. (By the way, my handle is @senarvi.)